lncfg/db.go

```diff
@@ -535,6 +560,21 @@ func (db *DB) GetBackends(ctx context.Context, chanDBPath,
 		}
 		closeFuncs[NSWalletDB] = sqliteWalletBackend.Close
 
+		var nativeSQLStore *sqldb.BaseDB
+		if db.UseNativeSQL {
+			nativeSQLiteStore, err := sqldb.NewSqliteStore(
+				db.Sqlite,
+				path.Join(chanDBPath, SqliteNativeDBName),
```

Comment:
Ah I see this now, I guess could also just modify the value directly (filepath join against the value in the config).